frontend/src/pages/Transaction.tsx

```diff
@@ -220,9 +220,13 @@ const Transaction: FC = () => {
                         </Grid>
                         {transaction.TxStatus === 'Not included' && (
                             <Grid>
-                            <Typography variant="body1" className="time-out-message">
-                                (check for inclusion time-out, please check gnosisscan)
-                            </Typography>
+                                <Typography variant="body1" className="time-out-message">
+                                (Waiting for inclusion timed out. Please check the block{' '}
+                                <Link href="https://gnosisscan.io/" target="_blank" rel="noopener noreferrer">
```

Comment:
Couldn't we directly link to the tx hash? I was more thinking of adding a linking symbol in the end.

Reply:
( you can also squash the two commits)